server/controllers/chartsController.js

```diff
@@ -102,6 +102,11 @@ export const graphDataController = async (dataDb, userAccess, chart_id) => {
           targetTotal: newTargetValue,
         }
       }
+      if (!!newTargetValue && !!studyTotals[constants.TOTALS_STUDY].targetTotal)
```

Comment:
if `newTargetValue` is `undefined` do we need to set the `targetTotal` to undefined? Similarly, if `targetTotal` is undefined should we not attempt to set it?

Reply:
I think we initially set `targetTotal` to undefined because we wanted to use the count instead of the target total when calculating percentages. Now it seems we want to change this functionality to behave more like the other studies.